sound/soc/sof/intel/hda-loader.c

```diff
@@ -470,3 +471,26 @@ int hda_dsp_post_fw_run(struct snd_sof_dev *sdev)
 	/* re-enable clock gating and power gating */
 	return hda_dsp_ctrl_clock_power_gating(sdev, true);
 }
+
+int hda_dsp_ext_man_get_cavs_config_data(struct snd_sof_dev *sdev,
```

Comment:
this function never returns an error. DOes it need to return an int? why not void?

Reply:
that's right, no return is required for ext manifest.